ToDoListTake2/Program.cs

```diff
@@ -88,7 +87,7 @@ static void Main(string[] args)
                 case 2:
                     var taskId = 0;
                     Console.WriteLine("What task id number would you like to update in your ToDo List? (Hint: enter the number of the id or type '0' to reference the list'");
```

Comment:
Yes it is essentially. I figured the user would not want to return to the main menu to view the list just to get the necessary id number, so I thought I would give them an option to view it after they clicked on "update". If there is a better way to give them the option to view it in order to get the id number, I would change it :D

Reply:
If 0 and 1 take the user to the same part of the todo list, I would make the instructions 1 to view the list.

Reply:
Oh I see. You're using 0 because the task id could be 1. What if you use a letter instead of the number 0 to view the list or maybe just display the list for the user here instead of giving them another number to reference list?

Reply:
I didn't want to use a letter because that opens up I would have to define what letter, make my app accept that letter and in various forms but not accept other letters. it just seemed easier to use 0, since 0 would never be used as the task id. I will definitely consider just displaying the list for the user when I refactor. I have a meeting with an instructor to get ideas on how to refactor and put more of the code into methods to clean up the program.cs :D